extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfigPersistenceUnit.java

```diff
@@ -84,6 +85,23 @@ public interface HibernateOrmRuntimeConfigPersistenceUnit {
     @ConfigDocMapKey("full-property-key")
     Map<String, String> unsupportedProperties();
 
+    /**
+     * Additional database object types to include in schema management operations.
+     *
+     * By default, Hibernate ORM only considers tables and sequences when performing
+     * schema management operations.
+     * This setting allows you to specify additional database object types that should be included,
+     * such as "MATERIALIZED VIEW", "VIEW", or other database-specific object types.
+     *
+     * The exact supported values depend on the underlying database and dialect.
+     *
+     * == Returns
+     * A list of additional physical table types to be managed by Hibernate ORM
+     *
+     * @asciidoclet
+     */
+    List<@WithConverter(TrimmedStringConverter.class) String> extraPhysicalTableTypes();
```

Comment:
As Marko [mentioned here](https://github.com/quarkusio/quarkus/pull/47921#discussion_r2095167741), and as I originally [advised there](https://github.com/quarkusio/quarkus/issues/47854#issuecomment-2882859315), this method needs to be placed in `HibernateOrmConfigPersistenceUnitSchemaManagement`, not in `HibernateOrmRuntimeConfigPersistenceUnit`.